src/public/media.ts

```diff
@@ -10,16 +10,19 @@ import {
   validateGetMediaInputs,
   validateViewImagesInput,
   validateScanBarCodeInput,
-  isMediaCallForVideoAndImageInputs,
+  isMediaCallSupportedOnMobile,
+  isVideoControllerRegistered,
+  isApiSupportedOnMobile,
 } from '../internal/mediaUtil';
+import { MediaControllerEvent, MediaControllerParam } from '../private/interfaces';
 import { sendMessageToParent } from '../internal/communication';
 import { registerHandler, removeHandler } from '../internal/handlers';
 import {
   captureImageMobileSupportVersion,
   mediaAPISupportVersion,
   getMediaCallbackSupportVersion,
   scanBarCodeAPIMobileSupportVersion,
-  videoAndImageMediaAPISupportVersion,
+  nonFullScreenVideoModeAPISupportVersion,
 } from '../internal/constants';
 
 export namespace media {
```

Comment:
Please ask @anandvnath or @SonalJha to review this PR.

Reply:
They are no longer part of the DeviceCapability team, @praxman, and @cpjoshi please review and approve if not done. Thanks.